backend/app/services/mongo/explore.py

```diff
@@ -1921,6 +1921,7 @@ async def fetch_flattened_tasks(
     limit: int = 1000,
     with_events: bool = True,
     with_sessions: bool = True,
+    with_removed_events: bool = False,
```

Comment:
Please add the following logic: if with_removed_events=True and with_events=False, then set with_events to True

Then document this behavior in the docstring